registry/auth/token/token_test.go

```diff
@@ -477,8 +475,8 @@ func TestAccessController(t *testing.T) {
 		t.Fatalf("accessControler did not get expected error - got %s - expected %s", challenge, ErrTokenRequired)
 	}
 
-	if authCtx != nil {
-		t.Fatalf("expected nil auth context but got %s", authCtx)
+	if grant != nil {
+		t.Fatalf("expected nil auth grant but got %s", grant)
```

Comment:
We need to change the formatter string here, though I wonder if it's worth implementing `Stringer` for `Grant` 🤔

Reply:
Good catch! `%s` would not have been terribly effective at formatting a `context.Context` legibly, either.